src/dapps/DappScreen.tsx

```diff
@@ -0,0 +1,367 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
```

Comment:
This file is also more or less just copy-pasted from `TabDiscover.tsx`, with only minor changes.